src/alchemtest/tests/__init__.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+class BaseDatasetTest:
+    # @pytest.fixture(scope="class",
+    #                 params = [(load_tyr2ala, ('forward', 'backward'), (1, 1)),
+    #                           ])
+    # def dataset(self, reques):
+    #   return super().dataset(request)
+
+    def dataset(self, request):
+        '''The input dataset is specified as:
+        load_method: The method for loading the dataset.
+        keys: The keys to the corresponding dataset.
+        value_length: The expected number of file for each key.
+        '''
+        load_method, keys, value_length = request.param
+        return load_method, keys, value_length
```

Comment:
If you come up with a nicer solution than the current one (which still keeps fixtures class-local) please make suggestions!!

Reply:
I think this is the way that most package uses so it looks good to me.